### PR TITLE
Helix reco single output + Track sequences reduction

### PR DESCRIPTION
**Pull request by gianipez**
- The helix finder algorithms now create a single output collection
- KinKal now merges the DxM with DxP and UxM with UxP (x=e, mu, pi)

### CODE REVIEW

**Comment by gianipez** [comment]
@FNALbuild run build test with https://github.com/Mu2e/Production/pull/371

**Comment by gianipez** [comment]
tagging: @dnbrow01 , @rlcee

**Comment by gianipez** [comment]
@FNALbuild run build test with https://github.com/Mu2e/Production/pull/371

**Comment by gianipez** [comment]
@FNALbuild run build test with https://github.com/Mu2e/Production/pull/371

File: Validation/src/ValKalSeed.cc
Comment (brownd1978) [inline Validation/src/ValKalSeed.cc:1]: Please increase the histogram version #

**Review by brownd1978 — APPROVED** [review]
This looks good. Have you run a validation test (ie Production/Validation/nightly/reco_00.fcl)? If not, could you please do so, and check that the results are compatible?

**Review by rlcee — APPROVED** [review]

**Comment by gianipez** [comment]
> This looks good. Have you run a validation test (ie Production/Validation/nightly/reco_00.fcl)? If not, could you please do so, and check that the results are compatible?

Hi Dave, I run successfully the `Validation/trigger.fcl` and `Validation/nightly/ceMix_00.fcl` jobs. Would that be enough?

File: Validation/src/ValKalSeed.cc
Comment (gianipez) [reply Validation/src/ValKalSeed.cc:1]: done

**Comment by gianipez** [comment]
@FNALbuild run build test with https://github.com/Mu2e/Production/pull/371

**Pull request merged**